package.json

```diff
@@ -29,11 +29,10 @@
   "homepage": "https://github.com/CanopyTax/node-jspm-jasmine#readme",
   "devDependencies": {
     "babel-cli": "^6.6.5",
-    "babel-preset-es2015": "^6.6.0"
-  },
-  "peerDependencies": {
-    "jasmine": "*",
-    "jspm": "*"
+    "babel-preset-es2015": "^6.6.0",
+    "commander": "^2.9.0",
+    "jasmine": "^2.4.1",
+    "jspm": "^0.17.0-beta.12"
```

Comment:
I originally listed jasmine and jspm as peerDependencies so that jspm/jasmine wouldn't be installed twice. Especially because that could result in a different version of jspm being used for executing tests than for bundling your app. I see in your commit messages that you moved them into devDependencies for npm3 compatibility, but npm3 still does support peerDependencies ([although it really does require that dependent module installs them on their own](https://docs.npmjs.com/files/package.json#peerdependencies)).

Perhaps for jasmine it makes sense to make it a true dependency (not a devDependency)? If so, I'd recommend using `latest` or maybe something like `>=2.0.0 <3.0.0` as the value of the dependency. That way we don't have to update this project every time jasmine bumps a minor version.

But for jspm, I'm thinking it would be best to leave it as a peerDependency to avoid the situation I was describing above. Curious on your thoughts on that, though.